### PR TITLE
set-ios-product-bundle-identifier 1.0.1

### PR DESCRIPTION
**Pull request by emenegro**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/set-ios-product-bundle-identifier/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+title: REPLACE PRODUCT_BUNDLE_IDENTIFIER
```
Comment (godrei) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: Please change this `REPLACE PRODUCT_BUNDLE_IDENTIFIER` to `Replace PRODUCT_BUNDLE_IDENTIFIER`

or even better would be: `Replaces Xcode's project PRODUCT_BUNDLE_IDENTIFIER`

We do not use uppercase letters (for regular words) in our step titles, see: https://www.bitrise.io/integrations

File: steps/set-ios-product-bundle-identifier/1.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+source_code_url: https://github.com/emenegro/bitrise-step-product-bundle-identifier
+support_url: https://github.com/emenegro/bitrise-step-product-bundle-identifier/issues
+published_at: 2017-03-28T12:48:46.457903822+02:00
+source:
+  git: https://github.com/emenegro/bitrise-step-product-bundle-identifier.git
+  commit: 83ff874f02ef39511c24a9ad239578035eec658b
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
```
Comment (godrei) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: please add:

- react-native
- cordova
- ionic

as well.

These project types also generates xcode project, support for these project types will come soon.

File: steps/set-ios-product-bundle-identifier/1.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- opts:
+    is_required: true
+    summary: Path to the iOS project `.xcodeproj` file.
+    title: Xcode project path
+  project_path: null
+- current_bundle_identifier: null
```
Comment (godrei) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: why do the step need the current bundle identifier, to set a new value?

File: steps/set-ios-product-bundle-identifier/1.0.1/step.yml (hunk 2 above)
Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: Ok.

File: steps/set-ios-product-bundle-identifier/1.0.1/step.yml (hunk 1 above)
Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: Ok

File: steps/set-ios-product-bundle-identifier/1.0.1/step.yml (hunk 3 above)
Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: I think of it as a simple replace, not a set. Which approach do you think is better?

Comment (viktorbenei) [inline steps/set-ios-product-bundle-identifier/1.0.1/step.yml]: From a user perspective it's way easier and less error prone if I don't have to specify a current value, only the value I want to set. So I'd definitely 👍 to not have to provide the current value, if that's possible 😉

**Comment by emenegro** [comment]
I close it, opened a new one: https://github.com/bitrise-io/bitrise-steplib/pull/933

**Pull request closed without merging**